src/etc/openbmp-mrt2bmp.yml

```diff
@@ -0,0 +1,71 @@
+# -------------------------------------------------------------------------
+# OpenBMP MRT2BMP configuration
+# -------------------------------------------------------------------------
+
+#
+# Collector - Where to send the BMP forwarded messages
+#
+collector:
+  # bmp-dev.openbmp.org
+  host: bmp-dev.openbmp.org  # Collector address
+  port: 5030     # Collector port number
+  delay_after_init_and_peer_ups: 5 # Delay in seconds to wait after sending the PEER_UP
+
+# Router views data config
+router_data:
+  router_directory_path: "route_views_sync/Router_Data/route-views.eqix.routeviews.org"   # Directory of the router which will be processed.
```

Comment:
Directory should not default not default to routeviews like this.  Instead it should default to something like: 'router_data/router1